spark/src/main/scala/io/delta/tables/execution/DeltaTableOperations.scala

```diff
@@ -26,6 +26,7 @@ import org.apache.spark.sql.delta.commands.{DeltaGenerateCommand, DescribeDeltaD
 import org.apache.spark.sql.delta.util.AnalysisHelper
 import org.apache.hadoop.fs.Path
 
+import org.apache.spark.sql.ColumnExtShim._
```

Comment:
If you only need `expr` importing `org.apache.spark.sql.ColumnExtShim` should be sufficient.